package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "description": "A starting point for the CSS Frameworks CA project",
   "scripts": {
-    "test": "echo \"We will learn more about testing in the Workflow course\" && exit 1"
+    "compile:sass": "node-sass sass/index.scss css/style.css -w"
```

Comment:
You don’t need node-sass here again. The script should be

```
"compile:sass": "sass --watch sass:dist/css && live-server"

```
The above script is saying SASS, take what is inside the sass folder, compile it into css style, and put them into the css folder in the dist folder.

Reply:
sorry i forgot to push changes. ive changed it to this and its working:
"watch": "sass --watch sass/index.scss:css/style.css & live-server"